regression/cbmc-sequentialization/Makefile

```diff
@@ -4,19 +4,19 @@ include ../../src/config.inc
 include ../../src/common
 
 ifeq ($(BUILD_ENV_),MSVC)
-GCC_ONLY = -X gcc-only
+POSIX_ONLY = -X requires_posix_only_headers
 else
-GCC_ONLY =
+POSIX_ONLY =
 endif
 
 test:
-	@../test.pl -e -p -c "../../../src/cbmc/cbmc --validate-goto-model --validate-ssa-equation" -X smt-backend $(GCC_ONLY)
+	@../test.pl -e -p -c "../../../src/cbmc/cbmc --validate-goto-model --validate-ssa-equation" $(POSIX_ONLY)
 
 test-cprover-smt2:
-	@../test.pl -e -p -c "../../../src/cbmc/cbmc --cprover-smt2" -X broken-smt-backend $(GCC_ONLY)
+	@../test.pl -e -p -c "../../../src/cbmc/cbmc --cprover-smt2" $(POSIX_ONLY)
 
 test-paths-lifo:
-	@../test.pl -e -p -c "../../../src/cbmc/cbmc --paths lifo" -X thorough-paths -X smt-backend -X paths-lifo-expected-failure $(GCC_ONLY)
+	@../test.pl -e -p -c "../../../src/cbmc/cbmc --paths lifo" $(POSIX_ONLY)
```

Comment:
:pick: It may be better to remove the `test-cprover-smt2` / `test-paths-lifo` sections from the `makefile`. The `CMakeLists.txt` doesn't do these and removing them for the  `makefile` would make it simpler.